app/models/mixins/miq_request_mixin.rb

```diff
@@ -143,4 +143,8 @@ def request_tags
     end
     ws_tag_data
   end
+
+  def workflow(prov_options = options, flags = {})
```

Comment:
@bzwei Looks like this is resolving the issue.  We should rename `prov_options` to `request_options` to match the mixin placement and this PR needs a test.